src/components/interfaces/MOBILE_API.xml

```diff
@@ -3038,6 +3094,12 @@
             <description>The short friendly name of the climate control module.
                 It should not be used to identify a module by mobile application.</description>
         </param>
+        <param name="currentTemperatureAvailable" type="Boolean" mandatory="false">
+              <description>
+                Availability of the reading of current temperature.
+                True: Available, False: Not Available, Not present: Not Available.
+        </description>
```

Comment:
Indentation is incorrect

Reply:
@jacobkeeler Fixed at commit fd9f349304e4d47d25d1fed79a0442a599717de8